drivers/soundwire/cadence_master.c

```diff
@@ -1923,13 +1924,20 @@ void sdw_cdns_config_stream(struct sdw_cdns *cdns,
 
 		if (cdns->bus.params.m_data_mode != SDW_PORT_DATA_MODE_NORMAL)
 			val |= CDNS_PORTCTRL_TEST_FAILED;
+	} else if (pdi->num == 0) {
+		val |= CDNS_PORTCTRL_BULK_ENABLE;
```

Comment:
double-check if this is correct, I wonder if pdi->num == 1 should also be dedicated to BULK.

Reply:
Yes, changed to `if (pdi->num == 0 || pdi->num == 1)`